src/Maestro/FeedCleanerService/FeedCleanerService.cs

```diff
@@ -61,52 +61,64 @@ public Task<TimeSpan> RunAsync(CancellationToken cancellationToken)
     [CronSchedule("0 0 2 1/1 * ? *", TimeZones.PST)]
     public async Task CleanManagedFeedsAsync()
     {
-        if (Options.Enabled)
+        if (!Options.Enabled)
         {
-            Dictionary<string, Dictionary<string, HashSet<string>>> packagesInReleaseFeeds =
-                await GetPackagesForReleaseFeedsAsync();
+            Logger.LogInformation("Feed cleaner service is disabled in this environment");
+            return;
+        }
+
+        Logger.LogInformation("Loading packages in release feeds...");
+
+        Dictionary<string, Dictionary<string, HashSet<string>>> packagesInReleaseFeeds =
+            await GetPackagesForReleaseFeedsAsync();
+
+        Logger.LogInformation("Loaded {versionCount} versions of {packageCount} packages from {feedCount} feeds",
```

Comment:
```suggestion
        Logger.LogInformation("Loaded {versionCount} versions of {packageCount} packages from {feedCount} release feeds",
```
Might be confused with the feeds we report below from specific orgs